packages/std-contracts/src/test/MudTest.t.sol

```diff
@@ -49,5 +50,6 @@ contract MudTest is DSTest {
     alice = utils.getNextUserAddress();
     bob = utils.getNextUserAddress();
     eve = utils.getNextUserAddress();
+    s.init(world, components);
```

Comment:
@alvrs I'm adding this here so that anyone using MudTest doesn't have to manually call `SystemStorage.init` in each test.